<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2" colspan="2">
    <widget class="QListWidget" name="blacklistedDates"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="blacklistButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Blacklist</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDateEdit" name="dateEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="phoneBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="emailBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="freeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Free this date</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="nameBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="3">
    <widget class="QStackedWidget" name="roleListStackedWidget"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameBox</tabstop>
  <tabstop>phoneBox</tabstop>
  <tabstop>emailBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>emailBox</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>emailEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phoneBox</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>phoneEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blacklistButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>blacklistDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>freeButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>freeDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameBox</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>nameEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nameEntered()</slot>
  <slot>phoneEntered()</slot>
  <slot>emailEntered()</slot>
  <slot>roleActivated(QListWidgetItem*)</slot>
  <slot>roleClicked(QListWidgetItem*)</slot>
  <slot>blacklistDate()</slot>
  <slot>freeDate()</slot>
 </slots>
</ui>
